### PR TITLE
Add example traces (plus typo in README)

### PR DESCRIPTION
**Pull request by mprimi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ check, followed by a tab, and then whether the history was valid. There are
 three validity states:
 
 - `true`      means the history was valid
-- `false`     means the history was valid
+- `false`     means the history was invalid
```
Comment (ligurio) [inline README.md:58]: Good catch, thanks!

File: histories/elle/rwregister-fail.edn [hunk 2]
```diff
@@ -0,0 +1,21 @@
+; P1 and P2 read X and see NULL
```
Comment (ligurio) [inline histories/elle/rwregister-fail.edn:1]: History is valid (but filename has opposite status):

```
sergeyb@pony:~/sources/elle-cli$ java -jar ./target/elle-cli-0.1.3-standalone.jar --model rw-register histories/elle/rwregister-fail.edn --verbose
{"valid?":true}
```

File: histories/elle/cas-fail.edn [hunk 3]
```diff
@@ -0,0 +1,13 @@
+; Two processes read initial value nil
```
Comment (ligurio) [inline histories/elle/cas-fail.edn:1]: Please add new histories to the `test.sh`.

**Review by ligurio — CHANGES_REQUESTED** [review]

File: histories/elle/cas-fail.edn [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+{:process 1, :type :invoke, :f :read, :value nil}
+{:process 1, :type :ok,     :f :read, :value nil}
```
Comment (ligurio) [inline histories/elle/cas-fail.edn:3]: Good and bad samples for CAS register are already exist, see `histories/knossos/cas-register/`.

File: histories/elle/cas-ok.edn [hunk 5]
```diff
@@ -0,0 +1,13 @@
+; Two processes read initial value nil
```
Comment (ligurio) [inline histories/elle/cas-ok.edn:1]: Good and bad samples for CAS register are already exist, see `histories/knossos/cas-register/`.

File: histories/elle/list-fail.edn [hunk 6]
```diff
@@ -0,0 +1,17 @@
+; P1 and P2 read x and see an empty list
```
Comment (ligurio) [inline histories/elle/list-fail.edn:1]: Bad sample for list-append is already exists, see `histories/elle/list-append-gh-30.edn`.

File: histories/elle/list-ok.edn [hunk 7]
```diff
@@ -0,0 +1,17 @@
+; P1 and P2 read x and see an empty list
```
Comment (ligurio) [inline histories/elle/list-ok.edn:1]: "Good" sample is already exist, see `histories/elle/paper-example.edn`.

File: histories/elle/rwregister-ok.edn [hunk 8]
```diff
@@ -0,0 +1,21 @@
+; P1 and P2 read X and see NULL
```
Comment (ligurio) [inline histories/elle/rwregister-ok.edn:1]: "Good" sample is already exist, see `histories/elle/rw-register.edn`.

File: histories/elle/rwregister-fail.edn [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+{ :process 1, :index 1, :type :invoke, :f :txn :value [[:r x]] }
```
Comment (ligurio) [inline histories/elle/rwregister-fail.edn:2]: Please fix a history sample and add it to the `test.sh`.

**Review by ligurio — CHANGES_REQUESTED** [review]

**Comment by mprimi** [comment]
I just wanted to share some samples I composed, thinking that more tests/examples are always better. 

Feel free to cherry pick or copy-paste whatever you think is good, if any, and throw away the rest.

**Comment by ligurio** [comment]
cherry-picked https://github.com/ligurio/elle-cli/commit/7970c5938baefa2669f1dc98bb2a497f008e950b
Thanks!

**Pull request closed without merging**